frontend/src/API/calls.ts

```diff
@@ -18,21 +26,33 @@ export const postSeason = (data: IPostApplicationSeason):
   return postJson(SEASON_URL, data);
 };
 
-export const getApplicationForm = (id: number):
-  PromiseLike<IApplication> => {
-  return getJson(`${GET_APPLICATION_BY_USERID_URL}${id}`);
+export const getApplicationFormBySelf = ():
+  PromiseLike<IApplicationForm> => {
+  return getJson(`${GET_APPLICATION_BY_SELF}`);
+};
+
+export const getApplicationFormByUsername = (username: string):
+  PromiseLike<IApplicationForm> => {
+  return getJson(`${GET_APPLICATION_BY_USERID_URL}${username}`);
 };
 
 export const getAllApplications = ():
-  PromiseLike<IApplication[]> => {
-  return getJson(GET_APPLICATIONS_URL);
+  PromiseLike<IApplicationForm[]> => {
+  return getJson(GET_ALL_APPLICATIONS);
```

Comment:
I suggest all url constants end in `URL` to lessen confusion with other constants